### ISSUE TITLE
Test Failure: io.openliberty.microprofile.metrics.internal.monitor_fat.TestEnableDisableFeaturesTest.testEDF5

### ISSUE THREAD

**Issue by fmhwong** [OpenLiberty/open-liberty#24590]
RTC 294706

```
testEDF5:java.lang.Throwable: 2023-02-11-10:21:17:074 There were 2 errors:
  java.io.IOException(Server returned HTTP response code: 500 for URL: https://localhost:8020/metrics?scope=vendor)
  java.lang.Exception(Errors/warnings were found in server TestEDF5 logs:
<br>[2/11/23, 10:21:11:341 PST] 0000002c com.ibm.ws.webcontainer.util.ApplicationErrorUtils           E SRVE0777E: Exception thrown by application class 'com.ibm.ws.session.http.AbstractHttpSession.isNew:218'
<br>[2/11/23, 10:21:11:572 PST] 0000002c com.ibm.ws.webcontainer.webapp                               E SRVE0315E: An exception occurred: java.lang.Throwable: java.lang.IllegalStateException: The method is called on an invalidated session: 58W_3izBZhmfEjRlEGi-Qg2)
at componenttest.custom.junit.runner.FATRunner.newThrowableWithTimeStamp(FATRunner.java:338)
at componenttest.custom.junit.runner.FATRunner.access$700(FATRunner.java:70)
at componenttest.custom.junit.runner.FATRunner$1.evaluate(FATRunner.java:304)
at componenttest.custom.junit.runner.FATRunner$2.evaluate(FATRunner.java:364)
at componenttest.custom.junit.runner.FATRunner.run(FATRunner.java:178)
  java.io.IOException(Server returned HTTP response code: 500 for URL: https://localhost:8020/metrics?scope=vendor)
  java.lang.Exception(Errors/warnings were found in server TestEDF5 logs:
<br>[2/11/23, 10:21:11:341 PST] 0000002c com.ibm.ws.webcontainer.util.ApplicationErrorUtils           E SRVE0777E: Exception thrown by application class 'com.ibm.ws.session.http.AbstractHttpSession.isNew:218'
<br>[2/11/23, 10:21:11:572 PST] 0000002c com.ibm.ws.webcontainer.webapp                               E SRVE0315E: An exception occurred: java.lang.Throwable: java.lang.IllegalStateException: The method is called on an invalidated session: 58W_3izBZhmfEjRlEGi-Qg2)
at componenttest.custom.junit.runner.FATRunner$1.evaluate(FATRunner.java:204)
```

**Comment by Channyboy** [comment]
Closed due to #24879

**Comment by Channyboy** [comment]
After discussion with session team, it was identified that it was a timing issue with the session "clean-up" thread executing it's periodic clean up processes right when the session thread is created. During sessions creation, the session time-out attribute is set to 0. It is in the latter process of the sessions creation process does the default of 30minutes is set. It is during the initial part of sessions creation that the separate thread which cleans up expired sessions executed and invalidated the "newly" created session and therefore causing an ISE for an invalidated session later in the session creation process.
